### PR TITLE
Add simple manual testing for the REST endpoints

### PR DESCRIPTION
**Pull request by TimDiekmann**
## 🌟 What is the purpose of this PR?

Currently, the REST API is not tested. Until we have proper end-to-end tests, we are going to test them manually

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1202542409311090/1202637673613186/f) _(internal)_

## 🚫 Soft-blocked by

The tests won't pass unless these PRs are merged

- #831 
- #832  

## 🔍 What does this change?

- Add an http-file to request the specified queries at the REST API
- Change the account id parameter for PUT operations `/data-type` and `/property-type` to match the rest

## ⚠️ Known issues

This is only a temporary solution and is not tested on CI.

## 🛡 What tests cover this?

None

## ❓ How to test this?

Start the binary and either run the HTTP file or run the queries manually

### CODE REVIEW

File: packages/graph/hash_graph/tests/rest-test.http [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+}
+
+> {%
+    client.test("status", function() {
+        client.assert(response.status === 200, "Response status is not 200");
+    });
+%}
+
+
+### Insert Person entity type
```
Comment (thehabbos007) [inline packages/graph/hash_graph/tests/rest-test.http:170]: This request doesn't pass on my end. I'll investigate

**Review by thehabbos007 — CHANGES_REQUESTED** [review]
It's almost working fully for me outside a jetbrains IDE, which is great!

File: packages/graph/hash_graph/tests/rest-test.http (hunk 1 above)
Comment (thehabbos007) [reply packages/graph/hash_graph/tests/rest-test.http:170]: Passes now, I had just had to pull :arrow_down:

File: packages/graph/hash_graph/tests/rest-test.http [hunk 2]
```diff
@@ -0,0 +1,239 @@
+### Insert Text data type
+POST localhost:3000/data-type
```
[the remark this answers is not in the document]
Comment (TimDiekmann) [reply packages/graph/hash_graph/tests/rest-test.http]: I added a comment at the top level to point to httpYac. I also added a job for `cargo-make` to run it 
```
cargo make test-rest-api
```

Currently, this is not running in CI as the binary has to run in parallel.

File: packages/graph/hash_graph/Makefile.toml [hunk 3]
```diff
@@ -19,3 +19,7 @@ private = false
 extend = "task"
 args = ["nextest", "run", "--cargo-profile", "${CARGO_MAKE_CARGO_PROFILE}", "@@split(CARGO_TEST_FLAGS, )", "--workspace", "--test", "integration", "--profile", "integration", "${@}"]
 dependencies = ["install-nextest"]
+
+[tasks.test-rest-api]
```
Comment (thehabbos007) [inline packages/graph/hash_graph/Makefile.toml:23]: Just for anyone else reviewing/looking at this. This isn't added to any CI, just a local dev command for `cargo make` for now :+1:

File: packages/graph/migrations/package.json [hunk 4]
```diff
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "dotenv-flow": "3.2.0",
+    "httpyac": "5.5.5",
```
Comment (thehabbos007) [inline packages/graph/migrations/package.json:17]: should be in `devDependencies` i think :)

**Review by thehabbos007 — DISMISSED** [review]
Thanks! I have one comment, but works great!

**Pull request merged**